Library/Categories/CoreData/NSPersistentStore+MagicalRecord.h

```diff
@@ -43,7 +43,8 @@
  */
 + (NSURL *) MR_fileURLForStoreNameIfExistsOnDisk:(NSString *)storeFileName;
 
-+ (NSURL *) MR_cloudURLForUbiqutiousContainer:(NSString *)bucketName;
++ (NSURL *) MR_cloudURLForUbiquitousContainer:(NSString *)bucketName;
++ (NSURL *) MR_cloudURLForUbiqutiousContainer:(NSString *)bucketName __deprecated;
```

Comment:
Same again — please use the macros that have been setup for this purpose: `MR_DEPRECATED_WILL_BE_REMOVED_IN_PLEASE_USE("MR_cloudURLForUbiquitousContainer:")` will tell the user _what_ they need to use instead of the deprecated method.